addons/mail/static/src/owl/components/composer/composer.scss

```diff
@@ -149,42 +166,52 @@
                 }
             }
         }
-
     }
 }
 
-.o_Composer_button {
+.o_Composer_button.o-open {
```

Comment:
Currently only emoji button is concerned by that, and it's based on its internal `state`.

The initial idea of that was to display it on any open tool button, like emoji and attachment buttons (greyed button while file manager was open, for instance). Maybe this could be a bit reworked to take this into account? Or just remove this if it takes too much time to adapt

Reply:
Rule was : 
```CSS
.o_Composer_button {
    &:focus {
        box-shadow: none;
        background-color: gray('200');
    }
    &.o-open {
        background-color: gray('200');
    }
}
```
I just rewrote it because I remove the `:focus` part.
Maybe we could do it in julien's PR or after it's merged ? Nothing to do with this PR purpose I think